ops/docker/op-stack-go/Dockerfile

```diff
@@ -101,6 +101,10 @@ ARG OP_SUPERVISOR_VERSION=v0.0.0
 RUN --mount=type=cache,target=/go/pkg/mod --mount=type=cache,target=/root/.cache/go-build cd op-supervisor && make op-supervisor  \
   GOOS=$TARGETOS GOARCH=$TARGETARCH GITCOMMIT=$GIT_COMMIT GITDATE=$GIT_DATE  VERSION="$OP_SUPERVISOR_VERSION"
 
+FROM --platform=$BUILDPLATFORM builder AS op-deployer-builder
+ARG OP_NODE_VERSION=v0.0.0
+RUN --mount=type=cache,target=/go/pkg/mod --mount=type=cache,target=/root/.cache/go-build cd op-chain-ops && make op-deployer  \
+  GOOS=$TARGETOS GOARCH=$TARGETARCH GITCOMMIT=$GIT_COMMIT GITDATE=$GIT_DATE VERSION="$OP_NODE_VERSION"
```

Comment:
Is the `OP_NODE_VERSION` intentionally passed here or is that just a copy/paste error?

All of the args passed to the `make op-deployer` command are not actually being read or used by the `go build` command downstream. In order to use them, the makefile would need to capture them as inputs and then pass them to the `go build` command. Here is how `op-node` does it [for example](https://github.com/ethereum-optimism/optimism/blob/feat/op-deployer-6/op-node/Makefile#L17-L29)

Reply:
that's a copy paste error - will fix. nice catch.